electricitymap/contrib/lib/models/events.py

```diff
@@ -296,8 +300,8 @@ def _validate_zone_key(cls, v):
         return v
 
     @validator("datetime")
-    def _validate_datetime(cls, v: dt.datetime, values: dict[str, Any]):
-        if v.tzinfo is None:
+    def _validate_datetime(cls, v: datetime, values: dict[str, Any]):
```

Comment:
The reason we imported datetime again as dt was because if we use `datetime` directly we get this issue:
```log
Variable not allowed in type expressionPylancereportInvalidTypeForm
(variable) datetime: Unknown
```
So if you revert that the other changes looks good and we can merge this.